Sources/Logto/Core/LogtoCore+Fetch.swift

```diff
@@ -82,7 +82,7 @@ public extension LogtoCore {
 
     struct RefreshTokenTokenResponse: Codable, Equatable {
         public let accessToken: String
-        public let refreshToken: String
+        public let refreshToken: String?
         public let idToken: String?
         public let scope: String
```

Comment:
This should be `scopes: String[]`, I think

Reply:
it's the api response, just raw string according to RFC

Reply:
Oops, sorry, my bad